### PR TITLE
Another(last one) batch of melee rework fixes

### PR DESCRIPTION
**Pull request by ErisMeleeEnjoyer**
## About The Pull Request
other UI format's block button no longer overlaps with others
no double tact on help intent, but any hit on help intent deals no damage whatsoever
raising weapon text adjustments
eswords don't have double tact

## Why It's Good For The Game
fixies

## Changelog
:cl: Kegdo
fix: raising weapon text adjustments
fix: other UI format's block button no longer overlaps with others
tweak: eswords don't have double tact
tweak: no double tact on help intent, but any hit on help intent deals no damage whatsoever
/:cl:

### CODE REVIEW

**Comment by DiaFRAME444** [comment]
> no double tact on help intent, but any hit on help intent deals no damage whatsoever

Please just remove double tact.

**Comment by ErisMeleeEnjoyer** [comment]
No.

**Comment by Alliostra** [comment]
How about you just make it check whether or not an item will be used for actually attacking something, and then make it raise the weapon only in that case? This seems like a really sloppy fix

**Comment by Iglpigl** [comment]
That would require him knowing how to code.

**Comment by Alliostra** [comment]
> That would require him knowing how to code.

dude? he's literally a staff developer??

**Comment by Iglpigl** [comment]
It was aquired in illegitimate ways, he gave Firefox 500 bad rats copies.

**Comment by Iglpigl** [comment]
Balance PR disguised as a bugfix.

**Comment by DiaFRAME444** [comment]
Quote from Firefox in the Discord

> Someone complained before that gibbing someone's head on accident help click is kind of dumb

made me realize why this is so bad, ironically. Someone's gonna forget to set off Help intent and get destroyed in a fight.

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 1]
```diff
@@ -272,6 +272,10 @@ meteor_act
 	if(!affecting)
 		return FALSE//should be prevented by attacked_with_item() but for sanity.
 
+	if(user.a_intent == I_HELP)
+		visible_message("<span class='danger'>[src] has been [pick("lightly poked", "tapped")] in the [affecting.name] with [I.name] by [user]!</span>")
```
Comment (SirRichardFrancis) [inline code/modules/mob/living/carbon/human/human_defense.dm:276]: We have contribution guidelines, try reading them, staff developer.
```suggestion
		visible_message(SPAN_DANGER("[src] has been [pick("lightly poked", "tapped")] in the [affecting.name] with [I.name] by [user]!"))
```

Comment (ErisMeleeEnjoyer) [reply code/modules/mob/living/carbon/human/human_defense.dm:276]: Sigmar guides my code, not some feeble guidelines

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 2]
```diff
@@ -272,6 +272,10 @@ meteor_act
 	if(!affecting)
 		return FALSE//should be prevented by attacked_with_item() but for sanity.
 
+	if(user.a_intent == I_HELP)
+		visible_message(SPAN_DANGER("[src] has been [pick("lightly poked", "tapped")] in the [affecting.name] with [I.name] by [user]!"))
```
Comment (Firefox13) [inline code/modules/mob/living/carbon/human/human_defense.dm:276]: Should it even be SPAN_DANGER?

**Comment by tristain01** [comment]
Add the armblade into thew melee system

**Comment by TheShown911** [comment]
Conflicts!

**Review by TheShown911 — APPROVED** [review]

**Pull request merged**